projects/demo/src/modules/services/notifications/notifications.component.ts

```diff
@@ -50,6 +50,7 @@ import {AlertExampleWithDataComponent} from './examples/4/alert-example-with-dat
     changeDetection,
 })
 export class ExampleTuiNotificationsComponent {
+    readonly method = import('!!raw-loader!./method.md');
     readonly exampleImportModuleComponent = exampleImportModuleComponent;
```

Comment:
Let's switch this whole file to `import` and `md`

Reply:
fixed